src/collections/components/CollectionForm/CollectionForm.tsx

```diff
@@ -87,6 +104,19 @@ export const CollectionForm: React.FC<
   const authorExternalId =
     collection.authors.length > 0 ? collection.authors[0].externalId : '';
 
+  // If this collection has any labels, let's keep track of them here
+  // to be able to show them in the form correctly
+  const [selectedLabels, setSelectedLabels] = useState<Label[]>(
+    // The API, via generated types, returns a `Maybe` type (Label [] | null),
+    // so type coalescing is required to keep the labels field happy.
+    (collection.labels as Label[]) ?? []
+  );
```

Comment:
Wondering if we define the state variable type as `<Label[] | null>` would help with type coalescing?